README.md

```diff
@@ -1,166 +1,171 @@
+import Vue from "vue";  
+  
+// import the plugin  
+import VueGAPI from "vue-gapi";  
+  
+// For all available scopes see here: https://developers.google.com/identity/protocols/googlescopes
 const apiConfig = {
-  apiKey: "<YOUR_API_KEY>",
-  clientId: "<YOUR_CLIENT_ID>.apps.googleusercontent.com",
-  discoveryDocs: ["https://sheets.googleapis.com/$discovery/rest?version=v4"],
-  scope: "https://www.googleapis.com/auth/spreadsheets"
-  // see all available scopes here: https://developers.google.com/identity/protocols/googlescopes'
+  apiKey: "<YOUR_API_KEY>",  
+  clientId: "<YOUR_CLIENT_ID>.apps.googleusercontent.com",  
+  discoveryDocs: ["https://sheets.googleapis.com/$discovery/rest?version=v4"],  
+  scope: "https://www.googleapis.com/auth/spreadsheets"  
 };
 
-// Use the plugin and pass along the configuration
-Vue.use(VueGAPI, apiConfig);
-```
+// Use the plugin and pass along the configuration  
+Vue.use(VueGAPI, apiConfig);  
+```  
 
-Exposes the `$getGapiClient` on the Vue instance that returns a promise containing the initialised instance of the Google API client.
-See full list of options for apiConfig object (here)[https://developers.google.com/identity/sign-in/web/reference] under gapi.auth2.SignInOptions
+For a complete list of the [apiConfig options](https://developers.google.com/identity/sign-in/web/reference) see the gapi.auth2.SignInOptions section.
 
-```html
+The plugin exposes `$gapi` on the Vue instance as entry point to the plugin functions and variables. The variable `$gapi.getGapiClient()` returns a promise containing the initialised instance of the Google API client.
+  
+```html  
 <script>
-
-export default {
-  name: 'my-component',
-
-  methods: {
+export default {  
+  name: 'my-component',  
+  methods: {  
     login () {
-      this.$getGapiClient()
-        .then(gapi => {
-          // gapi.sheets.spreadsheet.get(...)
-          // ...
-        })
+      this.$gapi.getGapiClient()  
+        .then(gapi => {  
+          // gapi.sheets.spreadsheet.get(...
+        }
+        .catch(err => {
+	      // Handle errors
+        }
     }
-  },
-
+  }
 }
-</script>
-```
-
-## Login
-
-This will shortcut the login process
-
-```html
-<script>
-export default {
-  name: 'login-shortcut',
-
-  methods: {
-    login () {
-      this.$login()
+</script>  
+```  
+  
+## Login  
+Initiate the login process.
+  
+```html  
+<script>  
+export default {  
+  name: 'login',
+  methods: {  
+    login () { 
+       this.$gapi.login()
+         .then(() => {
+           // Any actions needed after login
+         }
+         .catch(err => {
+	       // Handle errors
+         }
     }
   }
-
 }
-
 </script>
-```
-
-## Logout
-
-This will shortcut the logout process
-
-```html
-<script>
-export default {
-  name: 'logout-shortcut',
-
+```  
+  
+## Logout  
+Initiate the logout process.
+  
+```html  
+<script>  
+export default {  
+  name: 'logout',
   methods: {
-    logout () {
+    logout () { 
       this.$logout()
```

Comment:
this.$gapi.logout

Reply:
Yes you are right. Missed this one. Thanks for catching it.